src/modules/systemlib/circuit_breaker_params.c

```diff
@@ -106,7 +105,7 @@ PX4_PARAM_DEFINE_INT32(CBRK_AIRSPD_CHK);
  * @max 121212
  * @group Circuit Breaker
  */
-PX4_PARAM_DEFINE_INT32(CBRK_FLIGHTTERM);
+PARAM_DEFINE_INT32(CBRK_FLIGHTTERM, 121212);
```

Comment:
mc_att_control(_multiplatform) uses circuit breakers. That's why these params were changed. With this PR the ROS default value will be disconnected from the PX4/nuttx default value but it will continue to compile. The `PX4_PARAM_DEFINE_INT32` macro was a workaround to have only 1 definition of the default values.